apps/backend/src/location/entities/location.entity.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Entity,
+  Unique,
+  Column,
+  OneToMany,
+  Relation,
+  PrimaryColumn,
+} from 'typeorm';
+import { MerchantLocationEntity } from './merchant-location.entity';
+import { Merchant, NormalizedLocation } from '../../constants';
+
+@Entity('location')
+@Unique(['source_id', 'location_id'])
+export class LocationEntity {
+  @PrimaryColumn()
+  id: string;
+
+  @Column('varchar', { length: 15, nullable: false })
+  source_id: string;
+
+  @Column({ type: 'int4', nullable: false })
+  location_id: number;
+
+  @Column('varchar', { length: 255, nullable: false })
+  description: string;
+
+  @Column('varchar', { length: 10, nullable: false })
+  program_code: number;
+
+  @Column('varchar', { length: 255, nullable: false })
+  program_desc: string;
+
+  @Column('varchar', { length: 3, nullable: false })
+  ministry_client: number;
+
+  @Column('varchar', { length: 5, nullable: false })
+  resp_code: string;
+
+  @Column('varchar', { length: 5, nullable: false })
+  service_line_code: number;
+
+  @Column('varchar', { length: 4, nullable: false })
+  stob_code: number;
+
+  @Column('varchar', { length: 7, nullable: false })
+  project_code: number;
+
+  @OneToMany(() => MerchantLocationEntity, (merchant) => merchant.location, {
+    nullable: true,
+    cascade: true,
+  })
+  merchant_ids: Relation<MerchantLocationEntity[]>;
+
+  @Column()
```

Comment:
PT location id... should this be a separate entity? (for SBC we only use this if the payment_method.classification === 'CASH' so it is one to one and probably does not need to be split out - might not be the case for other ministries)